README: add suggestion to contribute with requested pages (#8165)
tldr-pages/tldr: README.md

- Adding your favorite command which isn't covered.
- Adding examples or improving the content of an existing page.

> - Searching our [help wanted](https://github.com/tldr-pages/tldr/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label for pages to make and contribute.

Author: ```suggestion
- Contributing requested pages from our issues with the [help wanted](https://github.com/tldr-pages/tldr/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label.
```

> - Searching our [help wanted](https://github.com/tldr-pages/tldr/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label for pages to make and contribute.

Reviewer: All of these work I think.
